strategies/outputs_file.py

```diff
@@ -1,15 +1,26 @@
-exec(open('header.py').read())
+from configparser import SafeConfigParser
 import os
-
+import sys
 
 def outpath_websource(websourcechoice):
+    parser = SafeConfigParser()
+    parser.read('./setup/user.ini')
+    candidate='OS'
+    print(parser.has_section(candidate))
+    user_OS=parser.get('OS', 'user_OS')
+    user_OS=user_OS[1:-1]
+    print (user_OS)
 
     if websourcechoice=='journal_en_francais_facile' or 'AZ_lyrics':
-        outpath_loc=output_dir1
         if user_OS=='Windows':
-            outpath = outpath_loc+'\\'+ websourcechoice +'\\'
-        if user_OS=='Ubuntu':
-            outpath = outpath_loc+'/'+websourcechoice +'/'
+            spacer_char='\\'
+        elif user_OS=='Ubuntu':
+            spacer_char='/'
+        else:
+            print('invalid OS')
+        outpath_loc=parser.get('file_locations', 'local_data_dir')
+        outpath_loc=outpath_loc[1:-1]
+        outpath = outpath_loc+spacer_char+ websourcechoice +spacer_char
```

Comment:
this whole thing can be os.path.join(output_loc.split(os.sep) + websourcechoice) + os.sep

Reply:
I got this:
TypeError: can only concatenate list (not "str") to list

because `output_loc.split(os.sep)` is a list 
and ` websourcechoice' isn't but  

this edit

`outpath = os.path.join(outpath_loc + os.sep+ websourcechoice + os.sep)`

 fixes that and keeps the spirit of using os. It relies on the user having typed their own path into the user.ini with the correct separator, but I think that's reasonable
`